source/frontend/src/components/common/List/SelectInput.tsx

```diff
@@ -19,6 +19,7 @@ interface ISelectInputProps<SelectTypes> {
     label: string;
     placeholder: string;
   }[];
+  autoSetting?: string;
```

Comment:
allows setting an attribute that should prevent browsers from trying to autofill. This makes sense for fields where we are already doing autocomplete like functionality, as sometimes the browser will draw its suggestions over our suggestions.